### PR TITLE
does compression interfere with socket.io?

### PR DESCRIPTION
**Pull request by bewest**
Test theory with @jimsiff.

### CODE REVIEW

**Comment by jimsiff** [comment]
:+1: fixed it for me on Azure.

File: server.js [hunk 1]
```diff
@@ -72,6 +72,11 @@ app.use(compression({filter: shouldCompress}));
 function shouldCompress(req, res) {
     //TODO: return false here if we find a condition where we don't want to compress
     // fallback to standard filter function
+    var IGNORE = /^\/socket/;
```
Comment (jasoncalabrese) [inline server.js]: before merging this back to dev we should undo the `IGNORE` attempt here

**Comment by jasoncalabrese** [comment]
@bewest I cherry-picked 8cebeb3f990af1ada04137f2cb842707e6e1fa9d to `dev`

**Comment by bewest** [comment]
Should fix tests, sorry. :-)

**Pull request merged**